frontend/assets/global.scss

```diff
@@ -58,15 +58,14 @@ button {
   background: $uiComponent;
   font-size: $buttonFormText;
   font-weight: 500;
-  
+
   transition: all .5s ease;
   border: none;
   outline: none;
   color: $fontColor;
 
-  // &:hover {
-  //     background: $uiComponentHighlighted;
-  // }
+  cursor: pointer;
+
```

Comment:
Warum wurde das enfernt?

Reply:
Weil das auskommentiert war und du in dem Meeting gestern meintest, hover sollte man nicht so oft benutzen weil das auf mobile nicht funktioniert. 
Ich hatte gedacht das brauchen wir nicht mehr.